scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -205,6 +206,19 @@ public Boolean isAddressReuseEnabled() {
 		return enableReuseAddress;
 	}
 
+	/**
+	 * Checks whether the connector should support the use of the TLS
+	 * <a href="https://tools.ietf.org/html/rfc6066#section-3">
+	 * Server Name Indication extension</a> in the DTLS handshake.
+	 * <p>
+	 * The default value of this property is {@code true}.
+	 * 
+	 * @return {@code true} if SNI should be used.
+	 */
+	public boolean isSniEnabled() {
```

Comment:
The getter is also intended to return a "Boolean" reference, so someone can test for null to check, if the default is intended to be used. You may check leshan builders to see, how it is intended to work.

Reply:
sure, oversaw this ...